mediaqueries/Overview.bs

```diff
@@ -1607,7 +1607,7 @@ Pointing Device Quality: the 'pointer' feature</h3>
 	Type: discrete
 	</pre>
 
-	The 'pointer' media feature is used to query about the presence and accuracy of a pointing device such as a mouse.
+	The 'pointer' media feature is used to query the presence and accuracy of a pointing device such as a mouse.
```

Comment:
This is for grammar, but also for consistency (if you check the rest of the spec, there are 5 instances of "query the ...", e.g. " query the value of individual features", "query the ability of the output device to modify the apearance of content ..." etc